dotnet-logging/AuditLogging/AuditLogApp/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using AuditLogApp.Data;
+using AuditLogApp.Models;
+using System.Text.Json;
+
+namespace AuditLogApp.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class ProductsController : ControllerBase
+{
+    private readonly ProductsDbContext _context;
+
+    public ProductsController(ProductsDbContext context)
+    {
+        _context = context;
+    }
+
+    // GET: api/Products
+    [HttpGet]
+    public async Task<ActionResult<IEnumerable<Product>>> GetProduct()
+    {
+        if (_context.Products == null)
```

Comment:
In some examples, you are using `is` when checking if something is `null` and on other places you are using `==`. Let's always use `is` for consistency 😄